src/Bicep.Core/PrettyPrint/DocumentBuildVisitor.cs

```diff
@@ -480,8 +483,23 @@ public override void VisitSyntaxTrivia(SyntaxTrivia syntaxTrivia)
                 this.Visit(syntax.Value);
             });
 
+        public override void VisitTupleTypeSyntax(TupleTypeSyntax syntax) =>
+            this.BuildWithConcat(() =>
+            {
+                this.Visit(syntax.OpenBracket);
+                this.VisitCommaAndNewLineSeparated(syntax.Children, leadingAndTrailingSpace: true);
+                this.Visit(syntax.CloseBracket);
+            });
+
+        public override void VisitTupleTypeItemSyntax(TupleTypeItemSyntax syntax) =>
+            this.BuildWithConcat(() =>
+            {
+                this.VisitNodes(syntax.LeadingNodes);
+                this.Visit(syntax.Value);
+            });
+
         public override void VisitUnionTypeSyntax(UnionTypeSyntax syntax) =>
-            this.Build(() => base.VisitUnionTypeSyntax(syntax), Spread);
+            this.BuildWithSpread(() => base.VisitUnionTypeSyntax(syntax));
```

Comment:
Just out of curiosity, how is the `BuildWithSpread()` different from `Build()` with `Spread` as a parameter?

Reply:
It isn't. I didn't see `BuildWithSpread` when I originally added `VisitUnionTypeSyntax` to this visitor